NeoMathEngine/src/CPU/x86/avx/src/BlobConvolution.inl

```diff
@@ -338,20 +338,19 @@ std::vector<int> CBlobConvolution<FltCnt>::getPixelOffsetSrcSteps( int srcDim, i
     }
     end++;
 
-    return vector<int>( start, end );
+    return std::vector<int>( start, end );
 }
 
 template<int FltCnt>
 void CBlobConvolution<FltCnt>::fillPixelOffset()
 {
-    using namespace std;
-    vector<int> pixelOffsetSrcStepsX = getPixelOffsetSrcSteps( SrcW, FltW, DilationW, StrideW, PaddingW );
-    vector<int> pixelOffsetSrcStepsY = getPixelOffsetSrcSteps( SrcH, FltH, DilationH, StrideH, PaddingH );
+    std::vector<int> pixelOffsetSrcStepsX = getPixelOffsetSrcSteps( SrcW, FltW, DilationW, StrideW, PaddingW );
```

Comment:
Why? It is inside of a function...

Reply:
For uniformity!
I have fixed everywhere, and also here. This is only function with that.
I know, that it limits the function area, however this is a 'using namaspace' in a header file. Not to stimulate use such constructions...
Should I revert this change?

Reply:
Not really. But other 2 comments are improtant.